### PR TITLE
Draft a new section on scaling

### PR DESCRIPTION
**Pull request by amcasey**
… that gives us a place for cross-linking and implementation notes

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/security/data-protection/configuration/scaling.md](https://github.com/dotnet/AspNetCore.Docs/blob/3505dc6fe0759b8208f5572fa059cfc1f8922299/aspnetcore/security/data-protection/configuration/scaling.md) | [aspnetcore/security/data-protection/configuration/scaling](https://review.learn.microsoft.com/en-us/aspnet/core/security/data-protection/configuration/scaling?branch=pr-en-us-33086) |

### CODE REVIEW

**Review by Rick-Anderson — APPROVED** [review]

File: aspnetcore/security/data-protection/configuration/scaling.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Configure ASP.NET Core Data Protection
```
Comment (Rick-Anderson) [inline aspnetcore/security/data-protection/configuration/scaling.md:2]:
```suggestion
title: Configure ASP.NET Core Data Protection in distributed or load-balanced environments
```

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]
My title , H2, and link suggestions should fix all the build errors.

**Comment by Rick-Anderson** [comment]
@amcasey  I don't understand the start of [Data Protection key management and lifetime in ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/security/data-protection/configuration/default-settings?view=aspnetcore-8.0):

If the app is hosted in [Azure Apps](https://azure.microsoft.com/services/app-service/), keys are persisted to the `%HOME%\ASP.NET\DataProtection-Keys` folder. This folder is backed by network storage a**_nd is synchronized across all machines hosting the app._**
* The DataProtection-Keys folder supplies the key ring to all instances of an app in a single deployment slot.
* Separate deployment slots (implies the a single deployment slot with many instances is OK)

File: aspnetcore/security/data-protection/configuration/scaling.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: Configure ASP.NET Core Data Protection in distributed or load-balanced environments
+author: acasey
+description: Learn how to configure Data Protection in ASP.NET Core for multi-instance apps.
+monikerRange: '>= aspnetcore-6.0'
```
Comment (amcasey) [inline aspnetcore/security/data-protection/configuration/scaling.md:5]: @Rick-Anderson How is this intended to be used.  Data Protection has been available and scalable for much longer than this, but most of the content in this article is for newer versions.

File: aspnetcore/security/data-protection/configuration/scaling.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+There are a variety of ways to make Data Protection work with multiple app instances. The most general to is explicitly establish a shared location for Data Protection keys. See <xref:host-and-deploy/scaling-aspnet-apps/scaling-aspnet-apps>, which shows how to configure a shared location for Data Protection keys using Azure Blob Storage.
+
+Newly created Azure Container Apps built using ASP.NET Core Kestrel 8.0 and later also benefit from automatic configuration. For more information see [Autoscaling considerations
+](/azure/container-apps/dotnet-overview#autoscaling-considerations).
+
+Azure Apps also support automatic key sharing.  For more information see <xref:security/data-protection/configuration/default-settings#key-management>.
```
Comment (amcasey) [inline aspnetcore/security/data-protection/configuration/scaling.md:22]: @Rick-Anderson Do we want to link to a product page or something for `Azure Apps`?

File: aspnetcore/security/data-protection/configuration/scaling.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+<!-- fixed: TODO: Make that an xref -->
+<!-- TODO: At this point, we could mention the environment variable for users who want to roll their own -->
```
Comment (amcasey) [inline aspnetcore/security/data-protection/configuration/scaling.md:25]: @claudiaregio What, if anything, did you want to say about the environment variable?

Comment (amcasey) [reply aspnetcore/security/data-protection/configuration/scaling.md]: Oops, I missed that Rick has already added some text.  Looks okay?

**Pull request closed without merging**